src/screens/Contact/contact.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./../../index.css">
+    <link rel="stylesheet" href="./contact.css">
+
+</head>
+
+<body>
+<nav>
+    <div>
+    <h4 class='logo'><a href='./../../index.html'>Cristian | Dev</a></h4>
```

Comment:
why are you linking to the html page 2 directories above?  isn't all your html files in the same directory?

Reply:
no I actually have a folder struct so src goes to screens and index files and screens have 2 directories contact and gallery than each of the files live in there.